src/main.ts

```diff
@@ -93,15 +125,46 @@ export default class ExecuteCodePlugin extends Plugin {
 	async saveSettings() {
 		await this.saveData(this.settings);
 	}
+	
+	/**
+	 * Adds run buttons to each open file. This is more robust and quicker than scanning
+	 * the entire document, even though it requires more iteration, because it doesn't
+	 * search the whole document.
+	 */
+	private iterateOpenFilesAndAddRunButtons() {
+		this.app.workspace.iterateRootLeaves(leaf => {
+			if (leaf.view instanceof FileView) {
+				this.addRunButtons(leaf.view.contentEl, leaf.view.file.path);
+			}
+		})
+	}
 
+	/**
+	 * Transform a language name, to enable working with multiple language aliases, for example "js" and "javascript".
+	 *
+	 * @param language A language name or shortcut (e.g. 'js', 'python' or 'shell')
+	 * @returns The same language shortcut for every alias of the language.
+	 */
+	private getLanguageAlias(language: string) {
```

Comment:
Was moved to TransformCode.ts, can be exported from that file and imported here instead of redefining?

Reply:
Thank you very much; done!